docs/gpu.md

```diff
@@ -29,6 +29,21 @@ Check your compute compatibility to see if your card is supported:
 |                    | Quadro              | `K2200` `K1200` `K620` `M1200` `M520` `M5000M` `M4000M` `M3000M` `M2000M` `M1000M` `K620M` `M600M` `M500M`  |
 
 
+### GPU Selection
+
+If you have multiple NVIDIA GPUs in your system and want to limit Ollama to use
+a subset, you can set `CUDA_VISIBLE_DEVICES` to a comma separated list of GPUs.
+Numeric IDs may be used, however ordering may vary, so UUIDs are more reliable.
+You can discover the UUID of your GPUs by running `nvidia-smi -L` If you want to
+ignore the GPUs and force CPU usage, use an invalid GPU ID (e.g., "-1")
+
+### Laptop Suspend Resume
+
+On linux, after a suspend/resume cycle, sometimes Ollama will fail to discover
+your NVIDIA GPU, and fallback to running on the CPU.  You can workaround this
+driver bug by reloading the NVIDIA UVM driver with `sudo rmmod nvidia_uvm &&
+sudo modprobe nvidia_uvm`
```

Comment:
Running this command gives me `rmmod: ERROR: Module nvidia_uvm is in use`

Reply:
Looks like your GPU could currently be in use by some process. Try running `nvidia-smi` to see if something is using it.

Reply:
xorg and ollama. 

But I'm confused as to what's the issue here, isn't this command supposed to stop nvidia drive and restart it (hence closing or restarting the apps using it too?)

Reply:
Let's get this merged, as it may help some folks.  @maxime1992 it sounds like in your scenario, a reboot may be necessary to reset the driver given other apps are still using it which is unfortunate.  Hopefully the underlying driver bug(?) will get resolved by NVIDIA soon.

Reply:
I believe that a more general solution involves specifying a different power mode for the card, as per https://askubuntu.com/questions/1228423/how-do-i-fix-cuda-breaking-after-suspend

Reply:
I had the same issue. 
I was able to reload the driver like this: 
```
sudo systemctl stop ollama
sudo rmmod nvidia_uvm && sudo modprobe nvidia_uvm
sudo systemctl start ollama
```